hydra-node/hydra-node.cabal

```diff
@@ -349,7 +349,7 @@ test-suite tests
     , plutus-ledger-api
     , process
     , QuickCheck
-    , quickcheck-dynamic
+    , quickcheck-dynamic                <2.0.0
```

Comment:
this formatting looks weird, I guess that's `cabal-fmt` but I am not keen on "tables" in text

Reply:
Yes. This was automatic by `cabal-fmt`